lookout/style/format/analyzer.py

```diff
@@ -64,24 +101,46 @@ def analyze(self, ptr_from: ReferencePointer, ptr_to: ReferencePointer,
                 fe = FeatureExtractor(language=lang, **rules.origin_config["feature_extractor"])
                 res = fe.extract_features([file], lines)
                 if res is None:
-                    comment = Comment()
-                    comment.file = file.path
-                    comment.confidence = 100
-                    comment.line = 1
-                    comment.text = "Failed to parse this file"
+                    if debug:
```

Comment:
How about enabling the debug features explicitly, for example `report_errorred_files` instead of `debug`